script/entrypoint

```diff
@@ -29,6 +29,8 @@ elif [ "$1" = "query" ]; then
     # to
     #     bq query [options] < FILE
     exec bq "${@:1:$#-1}" < "${@: -1}"
+elif [ "$1" = "copy_deduplicate" ]; then
+    exec "$(dirname $0)/copy_deduplicate" "${@:2}"
```

Comment:
instead of adding an alias here I think the docker command should be `docker run mozilla/bigquery-etl:latest script/copy_deduplicate [arg [arg...]]`, as supported in the `exec "$@"` below